### DISCUSSION TITLE
Do I need redis?

### DISCUSSION THREAD

**Discussion by berk76** [voiio/dramatiq-crontab#82] [Q&A] [answered]
Hello team,

from documentation I understand that redis is optional. But when I add `dramatiq_crontab` into `INSTALLED_APPS` and run `python manage.py migrate` I am getting error `No module named 'redis'`. So is redis mandatory then?

**Comment by berk76** [comment, marked as answer]
Now I see - redis is required by `dramatiq`. 
Question is answered :-)

**Reply by codingjoe** [reply]
You don't need it, but I'd recommend using it. You really want to avoid having two schedulers running at the same time.
